src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -54,8 +53,20 @@ public function include_payment_element() {
 
 		$payment_methods = ( new Merchant() )->get_payment_method_types();
 
-		// Don't load the Payment Element if just the Credit Card method is selected.
-		return ! ( 1 === count( $payment_methods ) && 'card' === $payment_methods[0] );
+		if ( 1 < count( $payment_methods ) ) {
+			return true;
+		} elseif ( 1 === count( $payment_methods ) && 'card' !== $payment_methods[0] ) {
+			return true;
+		}
+
+		/**
+		 * Filter to allow loading the Payment Element component
+		 *
+		 * @since TBD
+		 *
+		 * @param bool $include_payment_element Whether to include the Payment Element.
+		 */
+		return (bool) apply_filters( 'tec_tickets_commerce_stripe_include_payment_element', false );
```

Comment:
Might be worth passing either `$payment_methods` methods or the class object (`$this`) here

Reply:
Added, thanks!